.github/workflows/merge-develop-humble.yml

```diff
@@ -0,0 +1,62 @@
+name: Auto Merge develop to develop-humble
+
+on:
+  workflow_call:
+    inputs:
+      branch_from:
+        required: true
+        type: string
+      branch_to:
+        required: true
+        type: string
+      notify_team:
+        required: false
+        type: string
+        default: "DevOps"
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Git
+        run: |
+          git config user.name "SaikrishnaBairamoni"

```

Comment:
Nit: Should we create a generic devops account for STOL? 
So that even after all of us, it stays.

Reply:
Yeah, I think so but I was thinking this is an temporary change for specific humble changes

Reply:
Talked offline to handle this later.